modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc

```diff
@@ -0,0 +1,31 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+[id='configuring-managed-secure-boot-in-the-install-config-file_{context}']
+
+= Configuring managed Secure Boot in the `install-config.yaml` file (optional)
+
+To enable managed Secure Boot, add the `bootMode` configuration setting to each node.
+
+[source,yaml]
+.Example
+----
+hosts:
+  - name: openshift-master-0
+    role: master
+    bmc:
+      address: ipmi://<out-of-band-ip>
+      username: <user>
+      password: <password>
+    bootMACAddress: <NIC1-mac-address>
+    rootDeviceHints:
+     deviceName: "/dev/sda"
+    bootMode: UEFISecureBoot <1>
+----
+
+<1> The `bootMode` setting is `legacy` by default. Change it to `UEFISecureBoot` to enable managed Secure Boot.
+
+[NOTE]
+====
+See xref:configuring-nodes_ipi-install-prerequisites[Configuring nodes for Secure Boot manually] for additional details. to ensure the nodes can support managed Secure Boot. If not, you can xref:configuring-nodes-for-secure-boot_ipi-install-prerequisites[enable Secure Boot manually], which requires Redfish virtual media.
```

Comment:
You mean "Node requirements", not  "Configuring nodes for Secure Boot manually", right?

Reply:
and "Configuring nodes for Secure Boot manually" instead of "enable Secure Boot manually"